pkg/tenant/apikeys.go

```diff
@@ -225,6 +227,11 @@ func (s *Server) ProjectAPIKeyCreate(c *gin.Context) {
 		Modified:     key.Modified.Format(time.RFC3339Nano),
 	}
 
+	// Update project stats in the background
+	s.tasks.QueueContext(sentry.CloneContext(c), tasks.TaskFunc(func(ctx context.Context) error {
+		return s.UpdateProjectStats(ctx, key.ProjectID)
+	}), tasks.WithError(fmt.Errorf("could not update stats for project %s", key.ProjectID.String())))
```

Comment:
Only doing this in `ProjectAPIKeyCreate` and `ProjectTopicCreate` for right now so the status shows up properly in the frontend. I'm hoping that we can replace with the ensign topic once Tenant is setup for that.